hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/HibernateStatisticsTestSupport.java

```diff
@@ -99,8 +103,50 @@ protected void insertDummyEntities(int count, int childCount) {
         }
     }
 
+    protected void insertDummyNonStrictRWEntities(int count, int childCount) {
+        Session session = sf.openSession();
+        Transaction tx = session.beginTransaction();
+        try {
+            for (int i = 0; i < count; i++) {
+                DummyEntityNonStrictRW e = new DummyEntityNonStrictRW((long) i, "dummy:" + i, i * 123456d, new Date());
+                session.save(e);
+                for (int j = 0; j < childCount; j++) {
+                    DummyPropertyNonStrictRW p = new DummyPropertyNonStrictRW("key:" + j, e);
+                    session.save(p);
+                }
+            }
+            tx.commit();
+        } catch (Exception e) {
+            tx.rollback();
+            e.printStackTrace();
```

Comment:
I normally don't print stack traces. It can cause a lot of noise + your test doesn't fail if there is an unexpected exception. So either I deal with the exception or I propagate it. I would just propagate it .e.g have a look at ExceptionUtil. sneakyThrow or just wrap it in a RuntimeException.